strr-host-pm-web/app/components/RegistrationTermsConditions.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+const { t } = useNuxtApp().$i18n
+const { details } = storeToRefs(useConnectDetailsHeaderStore())
+const permitStore = useHostPermitStore()
+const { registration } = storeToRefs(permitStore)
+
+const conditionsOfApproval = computed(() => registration.value?.conditionsOfApproval)
+const { predefinedConditions, customConditions } = conditionsOfApproval.value || {}
```

Comment:
Would this break reactivity? Declaring predefinedConditions, customConditions separately with `computed` as a fix.

Reply:
Good catch, yea destructuring would break reactivity. However, this component does not depend on `conditionsOfApproval` changing dynamically. But I will update to keep reactive. Thanks!